docs/features/exporttopdf/README.md

```diff
@@ -0,0 +1,117 @@
+		return editor.editable().findOne( 'h1' ).getText() + '.pdf';
+	}
+
+The value is then calculated right before saving the file.
+
+### Output file configuration
+
+A number of options like output file format or margins can be set in the {@linkapi CKEDITOR.config.exportPdf_options CKEDITOR.config.exportPdf_options} object. It is sent to the service along with the HTML and CSS and processed on the server side. To check out the possible configurations, visit the [service documentation](https://pdf-converter.cke-cs.com/docs).
+
+### Relative vs absolute image URLs
+
+Images can be attached to the editor using relative URLs, but before data is sent to the service, such links are converted to absolute ones. In some cases it will mean that data will not be accessible by the server (e.g. if it is referenced locally or through the intranet) - remember to expose such assets publicly or use absolute URLs to publicly available assets. Other possibility is to use [base64 encoded images](https://pdf-converter.cke-cs.com/docs#section/Images/Insert-base64-encoded-image) - see the community plugin called [Base64 image](https://ckeditor.com/cke4/addon/base64image). Also {@linkapi CKEDITOR.config.baseHref CKEDITOR.config.baseHref} option may come in handy here to set the base path for editor assets to a different URL than editor itself.
+
+### Custom CSS rules
+
+Irrespectively from the editor type, inline rules are always preserved. There is however a difference in the behaviour according to the global editor styles.
+
+#### Classic editor
+
+Besides the inline styles, in order to mimic the editor content, all the default CKEditor 4 CSS rules are sent to the service. They can be overriden by {@linkapi CKEDITOR.config.contentsCss CKEDITOR.config.contentsCss} option or extended by {@linkapi CKEDITOR.addCss CKEDITOR.addCss} method.
+
+#### Divarea and Inline editor
+
+Here, due to the nature of these editors (as they use the styles of the webpage they are embedded in instead of editor's own rules), you should attach the additional stylesheet (one or more) using {@linkapi CKEDITOR.config.exportPdf_stylesheet CKEDITOR.config.exportPdf_stylesheet} configuration option. In such case these are the only non-inline styles that are sent.
+
+### Data preprocessing
+
+Plugin provides a custom {@linkapi CKEDITOR.editor#exportPdf exportPdf event}. It can be used for custom data processing (e.g. to ensure the output text will be black, not <span style="color:pink;background-color:yellow">pink on yellow background</span>). Editor uses it too, so just remember to add the right priority to the {@linkapi CKEDITOR.editor.on event listener}:
+
+* 1-14: Data is available in the original string format (it is extracted using {@linkapi CKEDITOR.editor.getData CKEDITOR.editor.getData} method).
+* 15: Data is preprocessed by the plugin: image relative paths are changed to absolute ones and editor's content is wrapped into a container with appropriate classes for styling.
```

Comment:
Do you think we could add some short info about the wrapper class name? So one could use correctly namespaced CSS rules which will be only applied for PDF file?

It won't fit here IMHO, but maybe somewhere in the Custom CSS section?